wiki/Community/en.md

```diff
@@ -0,0 +1,32 @@
+# Community
+
+This page lists ways for people from all around osu! to get in touch with each other. If you're looking for contribution opportunities, but don't know where to start, make sure to read [How you can help!](/wiki/Community/How_you_can_help!)
+
+## Places
+
+<!-- TODO: add forums -->
+
+- [Internet Relay Chat](/wiki/Community/Internet_Relay_Chat) (IRC)
+- [osu!dev Discord server](/wiki/Community/osu!dev_Discord_server)
+- Offline [organisations](/wiki/Community/Organisations)
+
+## Projects
+
+- [Community mentorship program](/wiki/Community/Community_Mentorship_Program)
+- [Mappers' Guild](/wiki/Community/Mappers_Guild)
+- [osu! wiki](/wiki/osu!_wiki)
+- [Project Loved](/wiki/Community/Project_Loved)
+
+## Initiatives
+
+- [osu! community meetings](/wiki/Community/osu!_Community_Meetings)
```

Comment:
file should be `osu!_community_meetings` now that i think about it

Reply:
there are other articles with seemingly incorrect casing now that i think about it, so you may ignore this since it should be its own issue

Reply:
renamed it anyway